aten/src/ATen/native/native_functions.yaml

```diff
@@ -189,6 +189,10 @@
 
 - func: _assert_tensor_metadata(Tensor a, SymInt[]? size=None, SymInt[]? stride=None, ScalarType? dtype=None) -> ()
 
+- func: print(str s) -> ()
```

Comment:
maybe `_print` for now, since we're just using this to test.

Before exposing print we want to think about what the API should be. i.e. is print(str) fine, or do we want some sort of `print("x: {}", x)`-like thing? The latter might not hold up as an op due to schema issues.